tfe/plugin_provider_test.go

```diff
@@ -55,14 +57,18 @@ func TestPluginProvider_providerMeta(t *testing.T) {
 			"hostname":        tftypes.NewValue(tftypes.String, tc.hostname),
 			"token":           tftypes.NewValue(tftypes.String, tc.token),
 			"ssl_skip_verify": tftypes.NewValue(tftypes.Bool, tc.sslSkipVerify),
-		}))
+		}),
+		)
+		if err != nil {
+			log.Printf("error: %v", err)
```

Comment:
Is this error fatal? I assume so since it would indicate that we passed an invalid value to NewDynamicValue ?

```suggestion
			t.Fatalf("Failed to create test config: %v", err)
```

Reply:
According to the [docs](https://pkg.go.dev/testing#B.Fatalf) `Fatalf` is equivalent to `Logf`, analogous to `Printf`.

Reply:
Right, but I guess my question is shouldn't the test run stop instead of just log if an error happens here?

Reply:
That's my understanding of the `Fatalf` method. It exits immediately after the test failure message is generated. Unlike `Printf` or `Errorf`.

You do agree the test should fail and exit, correct?